packages/algorand-js/src/interpreter/opcode-list.ts

```diff
@@ -151,3 +157,319 @@ export class Intc extends Op {
+    }
+  }
+}
+
+export class GreaterThan extends Op {
+  execute (stack: TEALStack): void {
+    this.assertStackLen(stack, 2);
+    const a = this.assertBigInt(stack.pop());
+    const b = this.assertBigInt(stack.pop());
+    if (a > b) {
+      stack.push(BigInt('1'));
+    } else {
+      stack.push(BigInt('0'));
+    }
+  }
+}
+
+export class LessThanEqualTo extends Op {
+  execute (stack: TEALStack): void {
+    this.assertStackLen(stack, 2);
+    const a = this.assertBigInt(stack.pop());
+    const b = this.assertBigInt(stack.pop());
+    if (a <= b) {
+      stack.push(BigInt('1'));
+    } else {
+      stack.push(BigInt('0'));
+    }
+  }
+}
+
+export class GreaterThanEqualTo extends Op {
+  execute (stack: TEALStack): void {
+    this.assertStackLen(stack, 2);
+    const a = this.assertBigInt(stack.pop());
+    const b = this.assertBigInt(stack.pop());
+    if (a >= b) {
+      stack.push(BigInt('1'));
+    } else {
+      stack.push(BigInt('0'));
+    }
+  }
+}
+
+export class And extends Op {
+  execute (stack: TEALStack): void {
+    this.assertStackLen(stack, 2);
+    const a = this.assertBigInt(stack.pop());
+    const b = this.assertBigInt(stack.pop());
+    if (a && b) {
+      stack.push(BigInt('1'));
+    } else {
+      stack.push(BigInt('0'));
+    }
+  }
+}
+
+export class Or extends Op {
+  execute (stack: TEALStack): void {
+    this.assertStackLen(stack, 2);
+    const a = this.assertBigInt(stack.pop());
+    const b = this.assertBigInt(stack.pop());
+    if (a || b) {
+      stack.push(BigInt('1'));
+    } else {
+      stack.push(BigInt('0'));
+    }
+  }
+}
+
+export class EqualTo extends Op {
+  execute (stack: TEALStack): void {
+    this.assertStackLen(stack, 2);
+    const a = this.assertBigInt(stack.pop());
+    const b = this.assertBigInt(stack.pop());
+    if (a === b) {
+      stack.push(BigInt('1'));
+    } else {
+      stack.push(BigInt('0'));
+    }
+  }
+}
+
+export class NotEqualTo extends Op {
+  execute (stack: TEALStack): void {
+    this.assertStackLen(stack, 2);
+    const a = this.assertBigInt(stack.pop());
+    const b = this.assertBigInt(stack.pop());
+    if (a !== b) {
+      stack.push(BigInt('1'));
+    } else {
+      stack.push(BigInt('0'));
+    }
+  }
+}
+
+export class Not extends Op {
+  execute (stack: TEALStack): void {
+    const a = this.assertBigInt(stack.pop());
+    if (a === BigInt('0')) {
+      stack.push(BigInt('1'));
+    } else {
+      stack.push(BigInt('0'));
+    }
+  }
+}
+
+// converts uint64 X to big endian bytes
+export class Itob extends Op {
+  execute (stack: TEALStack): void {
+    const stackValue = stack.pop();
+    const buf = encode(Number(stackValue));
+    const uint8arr = new Uint8Array(buf);
+    stack.push(uint8arr);
+  }
+}
+
+// converts bytes X as big endian to uint64
+// btoi panics if the input is longer than 8 bytes.
+export class Btoi extends Op {
+  execute (stack: TEALStack): void {
+    const uint8arr = stack.pop() as Uint8Array;
```

Comment:
```suggestion
    const uint8arr = this.assertBytes(stack.pop());
```
Also, should we change `uint8arr` to `const bytes`?